helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/service-headless.yaml

```diff
@@ -1,21 +1,28 @@
-{{- if .Values.global.serviceMonitor.enabled }}
+{{- if .Values.global.metrics.service.enabled }}
```

Comment:
this will create this service even when serviceMonitor is not enabled. is it intended?

Reply:
Yes. Monitoring agents that don't support serviceMonitor still need a ClusterIP service to hit to pick up metrics. This is my particular use case (https://docs.signalfx.com/en/latest/integrations/agent/monitors/prometheus-exporter.html)